packages/react-native/ReactAndroid/src/test/java/com/facebook/common/logging/FakeLoggingDelegate.kt

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+package com.facebook.common.logging
+
+import java.util.*
+
+class FakeLoggingDelegate : LoggingDelegate {
+  class LogLine(val priority: Int, val tag: String, val msg: String, val tr: Throwable?)
+
+  private var minLogLevel = FLog.VERBOSE
+  private val logs = ArrayList<LogLine>()
+
+  fun logContains(priority: Int, tag: String, throwMsg: String?): Boolean {
+    for (line in logs) {
+      if (matchLogQuery(priority, tag, throwMsg, line)) {
+        return true
+      }
+    }
+    return false
+  }
+
+  /** LoggingDelegate API */
+  override fun getMinimumLoggingLevel(): Int {
+    return minLogLevel
+  }
+
+  override fun setMinimumLoggingLevel(level: Int) {
+    minLogLevel = level
+  }
+
+  override fun isLoggable(level: Int): Boolean {
+    return level >= minLogLevel
+  }
+
+  private fun logImpl(priority: Int, tag: String, msg: String, tr: Throwable?) {
+    if (isLoggable(priority)) {
+      logs.add(LogLine(priority, tag, msg, tr))
+    }
+  }
+
+  override fun log(priority: Int, tag: String, msg: String) {
+    logImpl(priority, tag, msg, null)
+  }
```

Comment:
```suggestion
  override fun log(priority: Int, tag: String, msg: String) =
    logImpl(priority, tag, msg, null)
```

Please apply this to all the methods in this class